package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "brave-core",
-  "version": "1.73.79",
+  "version": "1.74.1",
```

Comment:
1.74.0 since we bump the minor first step in our public build pipeline.

Reply:
fixed